crates/astria-cli/README.md

```diff
@@ -0,0 +1,74 @@
+# Astria CLI
+
+Astria CLI is a command line tool for interacting with the Sequencer network
+and for managing your own rollup deployments. You can create a
+Sequencer account, check balances, generate rollup deployment configurations,
+deploy rollups, and more.
+
+## Dependencies
+
+- kubectl
+
+## Building
+
+```sh
+cargo build --release
+```
+
+## Running
+
+```sh
+# from monorepo root
+./target/release/astria-cli --help
+
+# examples:
+
+# create a rollup config
+./target/release/astria-cli rollup config create \
+  --use-tty \
+  --log-level DEBUG \
+  --rollup.name somerollupname \
+  --rollup.chain-id 0x1234 \
+  --rollup.network-id 42 \
+  --rollup.skip-empty-blocks \
+  --sequencer.initial-block-height 1 \
+  --sequencer.websocket ws://sequencer-service:26657/websocket \
+  --sequencer.rpc http://sequencer-service:26657 \
+  --rollup.genesis-accounts 0xaC21B97d35Bf75A7dAb16f35b111a50e78A72F30:100000000000000000000
+  
+# edit config
+./target/release/astria-cli rollup config edit \
+  --config somerollupname-rollup-config.yaml
+  <KEY> <VALUE>
+
+# delete config
+./target/release/astria-cli rollup config delete \
+  --config somerollupname-rollup-config.yaml
+
+# create deployment from config
+./target/release/astria-cli rollup deployment create \
+  --config somerollupname-rollup-config.yaml \
+  --faucet-private-key <FAUCET_PRIVATE_KEY> \
```

Comment:
what value should i be using for this? is this the private key to the rollup's faucet?